### PR TITLE
[IDE-189] fix: shortened plugin's name to just Snyk Security

### PR DESCRIPTION
**Pull request by Arvi3d**
### Description

This PR shortens the name of the plugin to just "Snyk Security"
This change is based on multiple complaints from the field and customers.

### Checklist

- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Snyk Security - Code and Open Source Dependencies Changelog
```
Comment (acke) [inline CHANGELOG.md:1]: Do we want to keep changelog?

"Snyk Security Changelog"

Comment (Arvi3d) [reply CHANGELOG.md:1]: yes, let me do it and update other plugins accordingly

Comment (Arvi3d) [reply CHANGELOG.md:1]: done

**Comment by Arvi3d** [comment]
@acke please re-review

**Review by bastiandoetsch — APPROVED** [review]

**Pull request merged**